lib/mumuki/classroom/locales/es-CL.yml

```diff
@@ -0,0 +1,3 @@
+---
+es-CL:
```

Comment:
Side note for the future: we should eventually rename `es` locales as `es-AR` locales. Or add both of them.

Reply:
Creating a separate issue for this so we don't lose track of it!